perf-dash.k8s.io/certificate.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: cert-manager.io/v1alpha2
+kind: Certificate
+metadata:
+  name: perfdash-k8s-io
+  namespace: perfdash
```

Comment:
This namespace is not present in `infra/gcp/namespaces/ensure-namespaces.sh`. Shouldn't this PR add it ?

Reply:
You are right! Good catch!

Reply:
Fixed! :-)